.github/workflows/mutation-testing.yml

```
@@ -0,0 +1,32 @@
+
+      - name: Cache pnpm dependencies
+        id: cache-pnpm-dependencies
+        uses: actions/cache@v4.2.3
+        with:
+          path: node_modules
+          key: ${{ runner.os }}-dependencies-${{ hashFiles('**/pnpm-lock.yaml') }}
+          restore-keys: |
+            ${{ runner.os }}-dependencies-
+
+      - name: Run mutation testing
+        run: make test-mutation
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)